components/Downloads/DownloadButton/index.module.css

```
@@ -1,5 +1,5 @@
 .downloadButton {
-  @apply justify-center;
+  @apply justify-center duration-300;
```

```suggestion
  @apply justify-center
    duration-300;
```

We separate tokens per line to ease readability 👀